fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubberTest.java

```diff
@@ -411,9 +412,13 @@ void testDetectDanglingUnnestedIndex() {
                         .setAllowRepair(false)
                         .build())
                 .build()) {
-            assertThrows(RecordDoesNotExistException.class, indexScrubber::scrubDanglingIndexEntries);
+            indexScrubber.scrubDanglingIndexEntries();
             indexScrubber.scrubMissingIndexEntries();
         }
+        Assertions.assertTrue(0 < timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_SCANNED));
```

Comment:
I think we can be more precise with this assertion. At the very least, this should be:

```
assertThat(timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_SCANNED), greaterThan(0));
```

To make the assertion a little more readable, but we actually know the expected number of records to scan should be `5 * numRecords`, I think

Reply:
✅